src/Databricks/custom/New-AzDatabricksWorkspace.ps1

```diff
@@ -61,7 +61,7 @@ function New-AzDatabricksWorkspace {
         [Microsoft.Azure.PowerShell.Cmdlets.Databricks.Category('Body')]
         [System.Management.Automation.SwitchParameter]
         # The value which should be used for this field.
-        ${EnableNoPublicIPValue},
+        ${EnableNoPublicIP},
```

Comment:
Except for here, you'll need also to add a directive in readme to rename EnableNoPublicIPValue to EnableNoPublicIP, otherwise `Az.Databricks.internal\New-AzDatabricksWorkspace @PSBoundParameters` will throw an error that `EnableNoPublicIP` is not recognized.

Reply:
I am sorry for this issue,I will fix it~thanks